action.yml

```diff
@@ -0,0 +1,11 @@
+name: "ServerlessTools"
+description: "Manage Serverless functions."
+inputs:
+  command:
+    description: "which command to run via the serverless-tools cli"
+    required: true
```

Comment:
IS the idea here that if (for whatever reason) we wanted to just deploy one function we could add it in the command input rather than having a second optional input (called `function` or something?)

Reply:
Yeah exactly that. So in the action you just use it like this: https://github.com/fac/event-system/pull/36/files#diff-9b0a49d1840887e3bd80347f2a20c2323881c4bcb9e7ccbccb40ebcbb54675c9R36-R38

I think it's more (easily) extendable that was. Essentially the CLI is the action, so if you're adding to it you just add your code the CLI and you get the action functionality for free.